animal_test.go

```diff
@@ -15,6 +15,18 @@ func TestNewGame(t *testing.T) {
 	}
 }
 
+func TestPlay(t *testing.T) {
+	t.Parallel()
+	testGame := animal.NewGame()
```

Comment:
another solution I just thought is to keep the variable global but receive it on the NewGame function. It would give a copy of the global variable to the function. I think it's a bit cleaner.

Reply:
marking this as resolved after Thiago's comment on slack.